src/lib/rest/HttpStatusCode.cpp

```diff
@@ -36,6 +36,7 @@ std::string httpStatusCodeString(HttpStatusCode code)
   case SccContextElementNotFound:            return "No context element found"; // Standard HTTP for 404: "Not Found"
   case SccBadVerb:                           return "Method Not Allowed";
   case SccNotAcceptable:                     return "Not Acceptable";
+  case SccConflict:                          return "TooManyResults";
```

Comment:
Ah, of course. I remember. 
We need one 'correct English phrase' and one 'enum-like string' ...